<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>618</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Python Projectile Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>681</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Python Projectile Simulator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>671</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Enter you Variables</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Angle of Projection (Theta) in degrees</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Initial Velocity (U0)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditVelocity">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>60</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>90</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxtheta">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m/s^2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Acceleration due to gravity (g)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_g">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>9</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>220</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>m/s^2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>140</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>degrees</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_simulate">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #1E90FF; /* DodgerBlue */
    color: white;
    border: none;
    padding: 8px 16px;
    font-size: 18px;
    border-radius: 4px;
}

QPushButton:pressed {
    background-color: #1C86EE; /* Darker blue when pressed */
    color: white;
}

QPushButton:hover {
    background-color: #1C86EE; /* Darker blue when hovered */
    color: white;
}

QPushButton:disabled {
    background-color: #A9A9A9; /* Light gray */
    color: #808080; /* Darker gray for text */
    border: none;
    padding: 8px 16px;
    font-size: 18px;
    border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string>Simulate Motion </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #f9c74f;  /* Yellowish color */
	font-size: 20px;
    color: white;
    border: none;
    padding: 5px 15px;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #f9a826;  /* Darker yellowish color */
}
QPushButton:pressed {
    background-color: #f8961e;  /* Even darker yellowish color */
}</string>
    </property>
    <property name="text">
     <string>Clear Values</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
